<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>atm</class>
 <widget class="QWidget" name="atm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ATM</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#atm{background-color: rgb(52, 80, 164);border-top-left-radius:15px;border-top-right-radius:5px;border-bottom-left-radius:15px;border-bottom-right-radius:5px}
</string>
  </property>
  <widget class="QPushButton" name="deposit_btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>111</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(225, 225, 225);}
QPushButton:hover{background-color: rgb(14, 122, 255);border:none;color:rgb(255, 255, 255);}
QPushButton:checked{background-color: rgb(20, 62, 134);border:none;color:rgb(255, 255, 255);}</string>
   </property>
   <property name="text">
    <string>存款</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cash_btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>111</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(225, 225, 225);}
QPushButton:hover{background-color: rgb(14, 122, 255);border:none;color:rgb(255, 255, 255);}
QPushButton:checked{background-color: rgb(20, 62, 134);border:none;color:rgb(255, 255, 255);}</string>
   </property>
   <property name="text">
    <string>取款</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>您好，请选择所需服务！</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="card_out_btn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(225, 225, 225);}
QPushButton:hover{background-color: rgb(255, 0, 0);border:none;color:rgb(255, 255, 255);}
QPushButton:checked{background-color: rgb(20, 62, 134);border:none;color:rgb(255, 255, 255);}</string>
   </property>
   <property name="text">
    <string>退卡 Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="transfer_btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>111</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(225, 225, 225);}
QPushButton:hover{background-color: rgb(14, 122, 255);border:none;color:rgb(255, 255, 255);}
QPushButton:checked{background-color: rgb(20, 62, 134);border:none;color:rgb(255, 255, 255);}</string>
   </property>
   <property name="text">
    <string>转账</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inquiry_btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>111</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(225, 225, 225);}
QPushButton:hover{background-color: rgb(14, 122, 255);border:none;color:rgb(255, 255, 255);}
QPushButton:checked{background-color: rgb(20, 62, 134);border:none;color:rgb(255, 255, 255);}</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pwchange_btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>320</y>
     <width>111</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(225, 225, 225);}
QPushButton:hover{background-color: rgb(14, 122, 255);border:none;color:rgb(255, 255, 255);}
QPushButton:checked{background-color: rgb(20, 62, 134);border:none;color:rgb(255, 255, 255);}</string>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="record_btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>111</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(225, 225, 225);}
QPushButton:hover{background-color: rgb(14, 122, 255);border:none;color:rgb(255, 255, 255);}
QPushButton:checked{background-color: rgb(20, 62, 134);border:none;color:rgb(255, 255, 255);}</string>
   </property>
   <property name="text">
    <string>交易记录</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-9</y>
     <width>511</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/back/back.png)</string>
   </property>
  </widget>
  <zorder>listView</zorder>
  <zorder>deposit_btn</zorder>
  <zorder>cash_btn</zorder>
  <zorder>label1</zorder>
  <zorder>card_out_btn</zorder>
  <zorder>transfer_btn</zorder>
  <zorder>inquiry_btn</zorder>
  <zorder>pwchange_btn</zorder>
  <zorder>record_btn</zorder>
 </widget>
 <resources>
  <include location="hnu.qrc"/>
 </resources>
 <connections/>
</ui>
